makefile

```diff
@@ -92,6 +92,21 @@ clean:
 package:
 	python -m build
 
+.PHONY:version_major
+version_major:
+	bump-my-version bump major
+	@echo Version number updated to `bump-my-version show current_version`
+
+.PHONY:version_minor
+version_minor:
+	bump-my-version bump minor
+	@echo Version number updated to `bump-my-version show current_version`
+
+.PHONY:version_patch
+version_patch:
+	bump-my-version bump patch
+	@echo Version number updated to `bump-my-version show current_version`
```

Comment:
[🐼] Could we also add these new commands to the list at the top
of this file so that `make help` show them?